processor/probabilisticsamplerprocessor/README.md

```diff
@@ -153,7 +153,7 @@ false, in which case erroneous data will pass through the processor.
 
 The following configuration options can be modified:
 
-- `sampling_percentage` (32-bit floating point, required): Percentage at which items are sampled; >= 100 samples all items, 0 rejects all items.
+- `sampling_percentage` (32-bit floating point, required): Percentage at which items are sampled; >= 100 samples all items, 0 accepts all items.
```

Comment:
My reading is: sampled == accepts items while not sampled == rejects, so IMO the original description is correct

Reply:
My bad, just tested it, you're right. `0` means all items are rejected.

Thanks!